src/shell.js

```diff
@@ -0,0 +1,14 @@
+import { exec } from 'child_process';
+
+/**
+ * @fileoverview Thin promise wrapper around ChildProcess#exec.
+ */
+export default function shell(command, options={}) {
```

Comment:
also see the `mz` package which provides promise friendly versions of exec/etc...